viewing_party/main.py

```diff
@@ -0,0 +1,221 @@
+
+def create_movie(title, genre, rating): 
+    movie = dict();
+    if title:
+        movie['title'] = title;
+    else:
+        return None
+    if genre:
+        movie['genre'] = genre;
+    else:
+        return None
+    if rating:
+        movie['rating'] = rating;
+    else:
+        return None
+    return movie
+
+
+def add_to_watched(user_data, movie):   
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+
+    watched_movie_temp_list = []
+
+    for key, values in user_data.items():
+        for movie_info in values:
+            if title == movie_info["title"]:
+                user_data["watchlist"].remove(movie_info)
+                watched_movie_temp_list.append(movie_info)
+    for watched_movie in watched_movie_temp_list:
+        user_data["watched"].append(watched_movie)
```

Comment:
You can clean this up a little:

```suggestion
    for movie_info in user_data["watchlist"]:
        if title == movie_info["title"]:
            user_data["watchlist"].remove(movie_info)
            user_data["watched"].append(movie_info)
```

You don't need to iterate through all of the keys/values since you just want the watchlist (like you did when you appended to the watched list).  You also can directly append to the watched list.